libsentrykube/service.py

```diff
@@ -41,24 +41,32 @@ def assert_customer_is_defined_at_most_once(
         )
 
 
-def set_service_paths(service_paths: List[str]):
-    for service_path in service_paths:
-        # Globbing is supported
-        paths = list(workspace_root().glob(service_path))
-        if not paths:
-            raise Exception(f"Invalid service path: {service_path}")
-        for path in paths:
-            if not path.is_dir():
-                continue
+def set_service_paths(service_paths: List[str], **namespaced_service_paths: List[str]):
+    targets = [
+        (None, service_paths),
+        *[(key, val) for key, val in namespaced_service_paths.items()],
+    ]
+    for collector_key, input_paths in targets:
+        collector = _services[collector_key] = _services.get(
+            collector_key, OrderedDict()
+        )
+        for service_path in input_paths:
+            # Globbing is supported
+            paths = list(workspace_root().glob(service_path))
+            if not paths:
+                raise Exception(f"Invalid service path: {service_path}")
+            for path in paths:
+                if not path.is_dir():
+                    continue
 
-            name = path.name
-            if name in _services:
-                raise Exception(f"Found duplicate service: {path}")
+                name = path.name
+                if name in collector:
+                    raise Exception(f"Found duplicate service: {path}")
 
-            if name == "customers":  # well-known; not an actual service
-                continue
+                if name == "customers":  # well-known; not an actual service
+                    continue
```

Comment:
Not related to your change, but this was an old ST thing that I believe we can now remove.
```suggestion
```